tests/components/bluesound/conftest.py

```diff
@@ -0,0 +1,60 @@
+"""Common fixtures for the Bluesound tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+from aiohttp import ClientConnectionError
+from pyblu import SyncStatus
+import pytest
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.bluesound.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture
+def mock_player_sync_status() -> Generator[AsyncMock, None, None]:
+    """Mock the sync status of a player."""
+    with patch(
+        "pyblu.Player.sync_status",
```

Comment:
Only mock at the place where we use this

Reply:
This means no fixture for this?

Reply:
No i mean, we should patch where we use this. So in stead of `pyblu.*`, do `homeassistant.components.bluesound.*`